.github/workflows/python-client.yml

```diff
@@ -28,22 +28,25 @@ on:
       - 'scripts/build-python-client.sh'
       - 'makefile'
       - '.github/workflows/python-tests/*'
+      - './magefiles/tests.go'
```

Comment:
Unrelated to the Scala client tests but I saw these discrepancies between `push.paths` and `pull_request.paths`.